common/app/conf/switches/CommercialSwitches.scala

```diff
@@ -461,6 +461,16 @@ trait PrebidSwitches {
     exposeClientSide = true,
   )
 
+  val prebidImproveDigitalSkins: Switch = Switch(
+    group = CommercialPrebid,
+    "prebid-improve-digital-skins",
+    "Serve Improve page skins via Prebid",
```

Comment:
For clarity can you write "Serve Collective/Improve page skins via Prebid" in the description. That way 2 years from now if it's still running but someone is asked to shut down the "Collective Skins" or the "Improve Skins" it will be clear this is that switch